### PR TITLE
Epic Hand Teleporter For Testing. 

### PR DESCRIPTION
**Pull request by potatoalienof13**
### Purpose
 <br>

To allow for testers, or admins to get around the station faster. 

### Notes:

I just copied the sprites from the remote signaler. 

### Changelog:

CL: [New] Hand teleporter for admins/testers.

### CODE REVIEW

**Comment by corp-0** [comment]
very nice!

File: UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using System;
+using UnityEngine; 
+
+/// <summary>
+/// Simple hand teleporter for convenience of admins.
+/// </summary>
+public class AdminHandTeleporter : MonoBehaviour, IInteractable<AimApply>
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs]: Could you change this to `ICheckedInteractable` and add the Default Interaction checks

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs]: well it is an admin tool after all

**Comment by ThatDan123** [comment]
Also tests will fail anyway as you need to commit the network manager changes as you added a new item.

File: UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs]: The Default Interaction checks stops other players from hack using it, as it check valid distance and inventory stuff

**Comment by potatoalienof13** [comment]
sad

**Comment by potatoalienof13** [comment]
Okay, tests passed in my repo.

**Review by ThatDan123 — APPROVED** [review]

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**